### PR TITLE
Collection of definitions and terms

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steven Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
@SteveLasker I installed PullApprove so you need to ensure you get 2 LGTMs from folks moving forward to merge changes in :)

**Comment by SteveLasker** [comment]
> @SteveLasker I installed PullApprove so you need to ensure you get 2 LGTMs from folks moving forward to merge changes in :)

Yup, thanks for getting everything setup.

File: definitions-terms.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* [Media Type](#media-type)
+* [OCI Image](#oci-image)
+* [Registry](#registry)
+* [YASS](#yass)
+
+## Artifact Author
+
+The owner of an artifact format. The [OCI Image Spec](https://github.com/opencontainers/image-spec/) is owned by the OCI working group. 
+
+An artifact is defined to be unique by it's `config.mediaType`.
```
Comment (stevvooe) [inline definitions-terms.md]:
```suggestion
An artifact is defined to be unique by its `config.mediaType`. 
```

File: definitions-terms.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+See [Registry](#registry)
+
+
+## Distribution Operator
+
+Vendors that implement and/or run the [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec/). 
+
+## Media Type
+
+The uniqueness of an artifact is defined by it's type. An artifact has a type, which has a collection of layers.
```
Comment (stevvooe) [inline definitions-terms.md]:
```suggestion
The uniqueness of an artifact is defined by its type. An artifact has a type, which has a collection of layers.
```

**Comment by mikebrow** [comment]
@caniszczyk The integration service -pull approve.. is pointing to the wrong group.. currently the image spec maintainers group. Can you please switch that over to artifacts group :-)

**Comment by caniszczyk** [comment]
@mikebrow should be fixed

**Comment by caniszczyk** [comment]
LGTM

**Pull request merged**